python/src/EnumerateFunctionImplementation_doc.i.in

```diff
@@ -190,9 +190,9 @@ strataIndex : int
 Returns
 -------
 cardinal : int
-    Number of members of the basis associated to the strata of index inferior or equal
+    Number of members of the basis associated to the strata of index lower or equal
     to *strataIndex*. In the context of product of polynomial basis, this is
-    the number of polynomials of the basis which total degree is inferior or
+    the number of polynomials of the basis which total degree is lower or
     equal to *strataIndex*.
```

Comment:
please drop EnumerateFunction changes if a squash is in order

Reply:
Ok. Moved to #2061